src/main/java/org/folio/circulation/rules/CirculationRulePolicyIdEntity.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.circulation.rules;
+
+import java.util.List;
+
+public class CirculationRulePolicyIdEntity {
```

Comment:
Can we think of a better name for this. What does this represent, is it a rule match?

Reply:
It represents policyId and conditions of current rule, could you please recommend a better name for that?

Reply:
I don't know, we are rather inventing terminology at this point. My first instinct suggests this is a circulation rule match.

Reply:
updated